FSSHTTPWOPIInspector/Test/WOPIautomation/WOPIautomation/App.config

```diff
@@ -3,31 +3,45 @@
   <!-- Note: Property values are case-insensitive unless otherwise specified.-->
   <appSettings>
     <!-- The browser used to access the site. The value should be one of following: IE32, IE64, Chrome, Firefox.-->
-    <add key="Browser" value="ie32"/>
-    <add key="UserName" value="contoso\wopiuser"/>
-    <add key="Password" value="|3wA7IcB#59De"/>
-    <add key="OtherUserName" value="Contoso\wopiuser1"/>
-    <add key="OtherPassword" value="|3wA7IcB#59De"/>
-    <add key="BaseAddress" value="http://sut02"/>
-    <add key="DocumentAddress" value="http://sut02/Shared%20Documents/Forms/AllItems.aspx"/>
-    <add key="Excel" value="..\..\..\Resource\Excel.xlsx"/>
-    <add key="Word" value="..\..\..\Resource\Doc.docx"/>
-    <add key="OneWithFileData" value="..\..\..\Resource\OneWithFileData.one"/>    
-    <add key="OneWithoutFileData" value="..\..\..\Resource\OneWithoutFileData.one"/>
-    <add key="NotebookTableOfContents" value="..\..\..\Resource\Open Notebook.onetoc2"/>
-    <add key="OneEncryption" value="..\..\..\Resource\Encryption.one"/>
-    
+    <add key="Browser" value="ie32" />
+    <add key="UserName" value="contoso\wopiuser" />
+    <add key="Password" value="|3wA7IcB#59De" />
+    <add key="OtherUserName" value="Contoso\wopiuser1" />
+    <add key="OtherPassword" value="|3wA7IcB#59De" />
```

Comment:
Github should not appear any password info

Reply:
Updated password infor in config